src/main/transports/queue.ts

```diff
@@ -1,19 +1,21 @@
-import { SentryRequestType } from '@sentry/types';
+import { TransportRequest } from '@sentry/types';
 import { logger, uuid4 } from '@sentry/utils';
 import { join } from 'path';
 
 import { readFileAsync, unlinkAsync, writeFileAsync } from '../fs';
 import { Store } from '../store';
-import { SentryElectronRequest } from './electron-net';
 
 const MILLISECONDS_PER_DAY = 86_400_000;
 
 interface PersistedRequest {
   bodyPath: string;
-  type: SentryRequestType;
```

Comment:
Note to self: This could require a coded migration in case of files existing in the queue when the SDK gets upgraded